src/styles/Carousel.scss

```diff
@@ -1,29 +1,33 @@
-.BrainhubCarousel {
-  overflow: hidden;
-  display: flex;
-  align-items: center;
+.BrainhubCarousel__container {
+  width: 100%;
 
-  &.BrainhubCarousel--isRTL {
-    direction: rtl;
-
-    .BrainhubCarousel__trackContainer {
-      .BrainhubCarousel__track {
-        direction: rtl;
+  .BrainhubCarousel {
```

Comment:
do we really need nesting everything and adding more specificity to rest selectors?

Reply:
Sure, we don't need it, but I don't think it hurts to add a little specificity.

Reply:
Hurts or not huts that is just bad practice. We need to correct it before merging. But the more important thing is about `flex-direction: row`.

Reply:
Fair enough, should be fixed now